src/renderer/screens/exchange/Swap2/Navbar/routes.json

```diff
@@ -1,5 +1,12 @@
 [
-  { "name": "swap", "path": "/swap", "title": "swap2.tabs.exchange" },
-  { "name": "history", "path": "/swap/history", "title": "swap2.tabs.history" },
-  { "name": "kyc", "path": "/swap/kyc", "title": "swap2.tabs.swap", "disabled": true }
-]
+  {
+    "name": "swap",
+    "path": "/swap",
+    "title": "swap2.tabs.exchange"
+  },
+  {
+    "name": "history",
+    "path": "/swap/history",
+    "title": "swap2.tabs.history"
+  }
+]
```

Comment:
new line

Reply:
My IDE annoyingly removes the last line break for JSON files, I just changed the behaviour now (hopefully) 🤞